lib/rails_translation_manager/locale_checker/missing_english_locales.rb

```diff
@@ -20,12 +20,23 @@ def format_missing_english_locales(keys)
   end
 
   def missing_english_locales
-    all_locales.each_with_object({}) do |locale, hsh|
-      missing_keys = exclude_plurals(locale[:keys]) - english_keys_excluding_plurals(all_locales)
+    all_locales.each_with_object({}) do |group, hsh|
+      next if group[:locale] == :en
+
+      keys = exclude_plurals(group[:keys])
+
+      missing_keys = keys.reject { |key| key_exists?(key) }
```

Comment:
Ah, nice suggestion! I was using `present?` to ensure `key_exists?` returns a boolean, which is expected of the method, rather than specifically testing for presence of an empty string.